Education_Pathways/frontend/src/components/Form.js

```diff
@@ -0,0 +1,374 @@
+import React, { Component } from "react";
+
+class Form extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      courseyear: "0",
+      division: "Any",
+      department: "Any",
+      campus: "Any",
+      searchterm: "",
+      minor: ""
+    };
+  }
+
+  handleCourseyearChange = (event) => {
+    this.setState({
+      courseyear: event.target.value
+    });
+  };
+
+  handleDivisionChange = (event) => {
+    this.setState({
+      division: event.target.value
+    });
+  };
+
+  handleDepartmentChange = (event) => {
+    this.setState({
+      department: event.target.value
+    });
+  };
+
+  handleCampusChange = (event) => {
+    this.setState({
+      campus: event.target.value
+    });
+  };
+
+  handleSearchtermChange = (event) => {
+    this.setState({
+      searchterm: event.target.value
+    });
+  };
+
+  handleMinorChange = (event) => {
+    this.setState({
+      minor: event.target.value
+    });
+  };
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    alert(`You have been submitted successfully!`);
+    console.log(this.state.courseyear);
+    console.log(this.state.campus);
+  };
+
+  render() {
+    return (
+      <form onSubmit={this.handleSubmit}>
+        <div style={{ marginTop: "3%" }}>
+          <label> Cour Year: </label>
+          <select
+            value={this.state.courseyear}
+            onChange={this.handleCourseyearChange}
+          >
+            <option selected value="0">
+              0
+            </option>
+            <option value="1"> 1</option>
+            <option value="2"> 2</option>
+            <option value="3"> 3</option>
+            <option value="4"> 4</option>
+          </select>
+        </div>
+        <div style={{ marginTop: "1%" }}>
+          <label> Division: </label>
+          <select
+            value={this.state.division}
+            onChange={this.handleDivisionChange}
+          >
+            <option selected value="Any">
+              Any
+            </option>
+            <option value="Faculty of Applied Science & Engineering">
+              Faculty of Applied Science & Engineering
+            </option>
+            <option value="Faculty of Arts and Science">
+              Faculty of Arts and Science
+            </option>
+            <option value="Faculty Of Music"> Faculty Of Music </option>
+            <option value="John H. Daniels Faculty of Architecture">
+              John H. Daniels Faculty of Architecture
+            </option>
+            <option value="University of Toronto Mississauga">
+              University of Toronto Mississauga
+            </option>
+            <option value="University of Toronto Scarborough">
+              University of Toronto Scarborough
+            </option>
+          </select>
+        </div>
+        <div style={{ marginTop: "1%" }}>
+          <label> Department: </label>
+          <select
+            value={this.state.department}
```

Comment:
I think you don't need to add the selections manually. I'll change this part in my pull request.